guides/common/modules/con_managing-dns-using-smart-proxies.adoc

```diff
@@ -0,0 +1,23 @@
+[id="Managing_DNS_Using_Smart_Proxies_{context}"]
+= Managing DNS Using {SmartProxies}
+
+{Project} can manage DNS records using your {SmartProxy}.
+DNS management contains updating and removing DNS records from existing DNS zones.
+Your {SmartProxy} can use multiple providers, allowing you to integrate with your existing DNS infrastructure or deploy a new one.
+
+After you have enabled DNS, your {SmartProxy} can manipulate any DNS server that complies with RFC 2136.
+
+.Available DNS Providers
+ifdef::orcharhino[]
+* `dns_dnscmd` - static DNS records in Microsoft Active Directory
+endif::[]
+* `dns_infoblox` - For more information, see {ProvisioningDocURL}Using_Infoblox_as_DHCP_and_DNS_Providers_provisioning[Using Infoblox as DHCP and DNS Providers] in _{ProvisioningDocTitle}_.
```

Comment:
Naming thing: in the installer we don't use the `dns_` prefix, but rather automatically prefix it. To be concrete: the user passes in `--foreman-proxy-dns-provider nsupdate_gss` to use the `dns_nsupdate_gss` provider. What should we follow here?

The same applies to DHCP.

Reply:
For me, this is fine as is because there are no `foreman-installer` commands in this concept file. I have checked to ensure that all foreman-installer commands are without `[dhcp-|dns-]` prefix.